src/uikit.cs

```diff
@@ -10429,8 +10816,21 @@ interface UIScrollView : UIFocusItemScrollableContainer {
 		[Export ("showsVerticalScrollIndicator")]
 		bool ShowsVerticalScrollIndicator { get; set; }
 
+		[TV (13,0), iOS (13,0)]
+		[Export ("verticalScrollIndicatorInsets", ArgumentSemantic.Assign)]
+		UIEdgeInsets VerticalScrollIndicatorInsets { get; set; }
+
+		[TV (13,0), iOS (13,0)]
+		[Export ("horizontalScrollIndicatorInsets", ArgumentSemantic.Assign)]
+		UIEdgeInsets HorizontalScrollIndicatorInsets { get; set; }
+
 		[Export ("scrollIndicatorInsets")]
-		UIEdgeInsets ScrollIndicatorInsets { get; set; } 
+		UIEdgeInsets ScrollIndicatorInsets {
+			[Deprecated (PlatformName.iOS, 13, 0, message: "Use 'VerticalScrollIndicatorInsets' or 'HorizontalScrollIndicatorInsets' instead.")]
+			[Deprecated (PlatformName.TvOS, 13, 0, message: "Use 'VerticalScrollIndicatorInsets' or 'HorizontalScrollIndicatorInsets' instead.")]
+			get;
+			set;
```

Comment:
weird, only the `getter` is deprecated ?

Reply:
Yup

```
@property(nonatomic)         UIEdgeInsets                 scrollIndicatorInsets; // use the setter only, as a convenience for setting both verticalScrollIndicatorInsets and horizontalScrollIndicatorInsets to the same value. if those properties have been set to different values, the return value of this getter (deprecated) is undefined.
- (UIEdgeInsets)scrollIndicatorInsets API_DEPRECATED("The scrollIndicatorInsets getter is deprecated, use the verticalScrollIndicatorInsets and horizontalScrollIndicatorInsets getters instead.", ios(2.0, 13.0), tvos(9.0, 13.0));
```